mica-webapp/src/main/resources/templates/search.ftl

```diff
@@ -272,6 +272,9 @@
 </div>
 <!-- ./wrapper -->
 
+<!-- MicaConfig in JSON Format -->
+<input type="hidden" id="mica-config" value='${configJson}'></input>
```

Comment:
if you remove the quotes you will have a JSON object directly: no need to parse it afterward.

Reply:
And why not making a const directly? What's the use of this input element?